python/res/simulator/simulation_context.py

```diff
@@ -30,6 +30,11 @@ def __init__(self, ert, sim_fs, mask, itr, case_data):
         self._ert.getEnkfSimulationRunner().createRunPath(self._run_context)
         self._sim_thread = self._run_simulations_simple_step()
 
+        # Wait until the queue is active before we finish the creation
+        # to ensure sane job status while running
+        while self.isRunning() and not self._queue_manager.isRunning():
```

Comment:
is there a case where the queue_manager is running and the thread is not alive?

Reply:
Most likely not. But there might be a case when neither the thread nor the queue_manager is running and then we should stop trying...